mmdet/core/bbox/assigners/approx_max_iou_assigner.py

```diff
@@ -63,19 +62,20 @@ def assign(self,
                approxs_per_octave,
                gt_bboxes,
                gt_bboxes_ignore=None,
-               gt_labels=None):
+               gt_labels=None,
+               background_label=-1):
```

Comment:
It is a bit risky to enable the users to set `background_label` since in most of cases it is -1.
So let's just fill the default `assigned_labels` as -1 and do not allow the users to set it.